### PR TITLE
Aggregate errors in xsd:import|include@schemaLocation for referenced grammar which have errors

### PR DESCRIPTION
**Pull request by angelozerr**
Aggregate errors in xsd:import|include@schemaLocation for referenced grammar which have errors

Fixes https://github.com/redhat-developer/vscode-xml/issues/596

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
This PR fixes 

 * redhat-developer/vscode-xml#596
 * when you use xsd:import and xsd:include with bad schemaLocation attribute value, now the error should highlight the schemaLocation attribute value
 * check some NPE that I discover when tests are executed (because in some test SharedSettings and languageclient can be null).

File: org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java [hunk 1]
```diff
@@ -176,45 +177,63 @@ public static Range toLSPRange(XMLLocator location, XMLSchemaErrorCode code, Obj
...
+				if (schemaLocationAttr != null) {
+					return XMLPositionUtility.selectAttributeValue(schemaLocationAttr);
+				}
+			} else {
+				//
+				// This error code occurs when an XSD file path is invalid in the following
+				// attributes:
+				// 1. xml-model href
+				// 2. xsi:schemaLocation
+				// 3.xsi:noNamespaceSchemaLocation
```
Comment (Mooninaut) [inline org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java:208]: Missing a space here

Comment (angelozerr) [reply org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java]: Thanks fixed

**Comment by angelozerr** [comment]
@rgrunber I have written test, you can review it now.

**Comment by rgrunber** [comment]
The change works for me, and I don't see any errors that are related so I would feel free to merge.

**Pull request merged**